src/overlays/actors/ovl_En_Dno/z_en_dno.c

```diff
@@ -122,8 +154,8 @@ void func_80A714B4(EnDno* this, PlayState* play) {
     do {
         actor = SubS_FindActor(play, actor, ACTORCAT_BG, ACTOR_BG_CRACE_MOVEBG);
         if (actor != NULL) {
-            if (ENDNO_GET_F(actor) == ENDNO_GET_F_1) {
-                Flags_SetSwitch(play, ENDNO_GET_7F0(actor));
+            if (BGCRACEMOVEBG_GET_F(actor) == 1) {
```

Comment:
I'm sure this will become clear when I look at the other PR, but it's a bit odd at the moment.

Reply:
Yeah, I didn't feel like creating a temporary enum for this when my other PR gives it a permanent name. I can do that if desired, though.